documentation/docs/scenarios/changeManagement.md

```diff
@@ -0,0 +1,343 @@
+```  
+[...]
+steps:
+  checkChangeInDevelopment:
+    changeManagement:
+      endpoint: 'https://example.org/cm'
+      [...]
+    failIfStatusIsNotInDevelopment: true
+```
+The parameters can also be provided when the step is invoked. See [Examples](#Examples).
+
+### Result
+
+* If the change document is in status "in development", the return value is `true`.
+* If the change document is not in status "in development", a `hudson.AbortException` is thrown.
+
+For exceptions, see **Exceptions** in [checkChangeInDevelopment](https://sap.github.io/jenkins-library/steps/checkChangeInDevelopment/).
+
+### Examples
+* All mandatory parameters are provided through the configuration and the `changeDocumentId` is retrieved from the Git commit history:
+```
+checkChangeInDevelopment script:this
+```
+* An explicit endpoint is provided and the `changeDocumentId` is searched for starting from the previous commit (`HEAD~1`):
+```
+checkChangeInDevelopment script:this,
+                         changeManagement: [
+                           endpoint: 'https:example.org/cm',
+                           git: [
+                             from: 'HEAD~1'
+                           ]
+                         ]
+```
+
+## (Optional) Create a Transport Request (`transportRequestCreate`)
+
+Create a transport request for a change document in SAP Solution Manager.
+
+### Context
+
+Depending on your workflow, this step is optional. If you already have a transport request, define it in the commit message. For example:
+
+```
+Lorem ipsum dolor sit amet, cum sucipat
+
+    sed diam nonumy eirmod tempor invidunt ut labore
+    et dolore magna aliquyam erat, sed diam voluptua.
+    At vero eos et accusam et justo duo
+    dolores et ea rebum nisi bene.
+
+    TransportRequest: ZZZDK900026
+```
+Per default, it is expected that one of the commits between origin/master and the HEAD branch contains this transport request ID.
+
+### Mandatory parameters
+
+| Parameter | Description |
+| --- | --- |
+| `script` | The common script envoronment of the running Jenkinsfile. The reference to the script that calls the pipeline step is privided by the `this` parameter, as in `script: this`. This allows the function to access the `commonPipelineEnvironment` to retrieve configuration parameters. |
+| `changeManagement/credentialsId` |  The ID of the credentials that are required to connect to SAP Solution Manager. |
+| `changeManagement/endpoint` | The address of SAP Solution Manager. |
+
+For an overview of the optional parameters, see **Parameters** in [transportRequestCreate](https://sap.github.io/jenkins-library/steps/transportRequestCreate/).
+
+### Step configuration
+
+The step is configured by using a customer configuration file that is provided as a resource in a custom shared library.
+```
+@Library('piper-library-os@master') _
+
+// the shared lib containing the additional configuration
+// must be configured in Jenkins
+@Library(foo@master') __
```

Comment:
The story with the `foo` lib is already explained above. Is it desired to repeat it?
Doesn't this make reading harder? Doesn't this making maintenance harder? But maybe there are good reasons for doing it like this from the docu-development point of view ...